selftests/unit/test_utils_service.py

```diff
@@ -55,17 +55,23 @@ def setUp(self):
             command_generator)
 
     def test_all_commands(self):
+        # Test all commands except "set_target" which is tested elsewhere
         for cmd, _ in ((c, r) for (c, r) in
                        self.service_command_generator.commands if
-                       c not in ["list", "set_target"]):
+                       c not in ["set_target"]):
```

Comment:
There's no need to keep using a list here, a simple equality check should be used.